lines.js

```diff
@@ -14,6 +14,24 @@ var historyForward = document.querySelector(".history-button.forward");
 var currentIndex = 0;
 var isOut;
 
+historyBack.buttonAppearance = function() {
+    if (currentIndex === projectHistory.length-1) {
+        this.classList.add("activated");
+    }
+    else if (currentIndex === projectHistory.length && this.classList.contains("activated")) {
+        this.classList.remove("activated");
+    }
+}
+
+historyForward.buttonAppearance = function() {
+    if (currentIndex === 1 && !this.classList.contains("activated")) {
```

Comment:
So, in case we call it with `currentStep === 2`, we do not need to activate button?